proposals/benefits_of_lc.md

```diff
@@ -1,3 +1,13 @@
-## Paper to discuss benefits of LC for primarily library administrators
+## Overview of Library Carpentry for Library Admin
 
-...
+(Based on [this document](https://github.com/LibraryCarpentry/governance/blob/master/proposals/lir-heanet-pager.md))
+
+## What is Library Carpentry?
+Digital technology is fundamentally altering academic and research libraries. The rapid pace of change has left a gap in the skill set of library staff. [Library Carpentry](https://librarycarpentry.org/) was founded to fill this gap by nurturing the new skills and competencies required for the operation of a 21st century research library. It is a growing community of instructors and lesson developers whose mission is to teach librarians the tools, techniques, and best practices to: work with data, automate repetitive tasks, support world-class research, and deliver data-savvy, team-based services to the academic and wider community.
+
+Using the pedagogical practices of live coding, pair programming, discussion and exercises, [Library Carpentry workshops](https://librarycarpentry.org/lessons/) create a safe and collaborative space for important concepts in computing and data, including data manipulation and organization, text and data mining, and workflow versioning and management. Not only do these skills help librarians create reproducible workflows and reliable operations for data-centric tasks, they give librarians a common language with researchers that can lead to a better mutual understanding of data issues and it paves the way to greater collaboration between the library and their research community.
+
+Since 2015, Library Carpentry has held [over 75 workshops worldwide](https://librarycarpentry.org/past_workshops/) and three global sprints involving hundreds of contributors to improve the lesson materials. Librarians who have attended workshops have reported improved abilities in automating curation workflows, communicating more effectively with researchers, collaborating on projects, and generating reports more efficiently, to name a few. (add more examples from Elizabeth here). 
```

Comment:
The workshop count is not accurate. It was 142 total workshops in November which you can see here https://github.com/LibraryCarpentry/librarycarpentry.github.io/blob/06cb6f82286006c8f1b860c76f9554dcfec6cbe3/_data/workshops.yml @maneesha can this be fixed?

Reply:
I will update the number and remove the link for now